src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataReaderTest/DataReaderTest.cs

```diff
@@ -65,7 +64,25 @@ public static void MultiQuerySchema()
             }
         }
 
+        [CheckConnStrSetupFact]
+        public static void GetSchemaTable_returns_null_when_no_resultset()
```

Comment:
Test name is inversed :)

Reply:
Oh!

Reply:
Another possible note is the returned table should have no rows, but should still have the columns it has when there are rows.